src/asmcnc/skavaUI/screen_file_loading.py

```diff
@@ -240,6 +243,11 @@ def objectifiled(self, job_file_path, dt):
             self.sm.pm.show_error_popup(file_empty_warning)
             self.sm.current = "local_filechooser"
             return
+        if self.model_manager.is_machine_drywall():
+            if not self.dwt_file_id_line in self.job_file_as_list:
```

Comment:
Just a suggestion: In the interest of speed, can we limit the search to the first (x) number of lines maybe?

Reply:
Is the ID going to be guaranteed to be at the top?

Reply:
Also wanted to add, that since this line is only executed on drywall machines, on which only .dwt files are allowed to be loaded, this will only be slow in the case that somebody uses a non valid .dwt file, which should be rare?

Reply:
Yeah that's reasonable